core/templates/pages/oppia-root/routing/access-validation-backend-api.service.ts

```diff
@@ -58,6 +58,9 @@ export class AccessValidationBackendApiService {
   COLLECTION_PLAYER_PAGE_ACCESS_VALIDATOR_URL_TEMPLATE =
     '/access_validation_handler/can_access_collection_player_page/<collection_id>'; // eslint-disable-line max-len
 
+  COLLECTION_EDITOR_PAGE_ACCESS_VALIDATOR =
+    '/access_validation_handler/can_access_collection_editor_page/<collection_id>'; // eslint-disable-line max-len
+
```

Comment:
```suggestion
  COLLECTION_EDITOR_PAGE_ACCESS_VALIDATOR = (
  
        '/access_validation_handler/' +
        'can_access_collection_editor_page/<collection_id>'); 

```

Avoid adding a bypass flag wherever possible!

Reply:
fixed